app/models.py

```diff
@@ -2068,6 +2068,14 @@ class FactNotificationStatus(db.Model):
     updated_at = db.Column(db.DateTime, nullable=True, onupdate=datetime.datetime.utcnow)
 
 
+class FactProcessingTime(db.Model):
+    __tablename__ = "ft_processing_time"
```

Comment:
the other ft tables have a `created_at` and `updated_at`. Do we want these? Maybe not the updated_at, but the created_at might have merit, given we could feasibly re-generate rows at some point. we added the created_at and updated_at in response to issues we had debugging stuff in the other ft tables - https://github.com/alphagov/notifications-api/pull/1884

Reply:
It's easy enough to add those columns.